src/Workspaces/CSharp/Portable/Rename/CSharpRenameRewriterLanguageService.cs

```diff
@@ -522,61 +425,304 @@ private async Task<SyntaxToken> RenameAndAnnotateAsync(SyntaxToken token, Syntax
                 return result;
             }
 
-            private bool IsRenameLocation(SyntaxToken token)
+            public override SyntaxTrivia VisitTrivia(SyntaxTrivia trivia)
+            {
+                var newTrivia = base.VisitTrivia(trivia);
+                // Syntax token in structure trivia would be renamed when the token is visited.
+                if (!trivia.HasStructure && _stringAndCommentRenameContexts.TryGetValue(trivia.Span, out var textSpanRenameContexts))
+                {
+                    var subSpanToReplacementTextInfo = CreateSubSpanToReplacementTextInfoDictionary(textSpanRenameContexts);
+                    return RenameInCommentTrivia(trivia, subSpanToReplacementTextInfo);
+                }
+
+                return newTrivia;
+            }
+
+            public override SyntaxToken VisitToken(SyntaxToken token)
```

Comment:
This should match the logic of the old `VisitToken` method.
Except it is expanded to rename and annotate multiple symbols